### PR TITLE
consistent wording; stream ops are "operators"

### PR DESCRIPTION
**Pull request by ktoso**
Refs https://github.com/akka/akka/issues/24995 but does not address everything that that ticket mentions

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: akka-docs/src/main/paradox/stream/stream-customize.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ or the downstreams. Even for stages that do not complete or fail in this manner,
 
 @@@ div { .group-scala }
 
-## Extending Flow Combinators with Custom Operators
+## Extending Flow operators with Custom Operators
```
Comment (patriknw) [inline akka-docs/src/main/paradox/stream/stream-customize.md]: capital O, or rewrite all headers in this page to lower, which is our recommended style

**Review by patriknw — APPROVED** [review]
LGTM, with one nitpick

File: akka-docs/src/main/paradox/stream/stream-customize.md (hunk 1 above)
Comment (ktoso) [reply akka-docs/src/main/paradox/stream/stream-customize.md]: Ah thanks, so `Extending Flow operators with custom operators`?

Comment (ktoso) [reply akka-docs/src/main/paradox/stream/stream-customize.md]: Made it `## Extending Flow Operators with Custom Operators
`

**Pull request merged**